lib/medium-admin.php

```diff
@@ -790,11 +778,35 @@ public static function cross_post($post, $medium_post, $medium_user) {
       $path = "/v1/users/{$medium_user->id}/posts";
     }
 
-    $created_medium_post = self::_medium_request("POST", $path, $medium_user->token, $data, array(
-      "Content-Type" => "application/json"
-    ));
+    try {
+      $created_medium_post = self::_medium_request("POST", $path, $medium_user->token, $data, array(
+        "Content-Type" => "application/json"
+      ));
+    } catch (Exception $err) {
+      // Retry once if we got a timeout
+      if ($err->getCode() == 1001) {
```

Comment:
lets use a constant if we're defining our own error code. Which, by the way, you should use -2, because 1001 already has a meaning.